ext/rotoscope/rotoscope.c

```diff
@@ -180,6 +180,7 @@ static void event_hook(VALUE tpval, void *data)
   if (in_fork(config))
   {
     rb_tracepoint_disable(config->tracepoint);
+    config->state = RS_OPEN;
```

Comment:
I wouldn't really want to rely on this line for `rotoscope.closed?`, since then we are relying on a call being made after forking for the desired behaviour of `rotoscope.closed?`.  I added this line mostly so that we can check if we are tracing during garbage collection, where we can't check the tracepoint object there in case it has already been garbage collected.